src/frontier/frontier_cmd_arcade.c

```diff
@@ -0,0 +1,526 @@
+            } else {
+                if (BattleArcade_MultiplayerCheck(arcadeCtx->type) == FALSE) {
+                    profile = Save_PlayerData_GetProfileAddr(param->saveData);
+                } else {
+                    profile = sub_02034818(var1);
+                }
+                BufferPlayersName(ctx->frontierSystem->unk44, var0, profile);
+            }
+            break;
+        case 39:
+            arcadeCtx->unkA7A = var1;
+            ov80_02233F1C(ctx, arcadeCtx, var0);
+            break;
+        case 40:
+            ov80_0222A474(&arcadeCtx->unkF4[0], arcadeCtx->unk74[arcadeCtx->unk11], 0xb, 0xcc);
+            ov80_0222A474(&arcadeCtx->unkF4[1], arcadeCtx->unk74[arcadeCtx->unk11 + 7], 0xb, 0xcc);
+            break;
+        case 41:
+            ov80_02234B7C(arcadeCtx, frontierMap, var0, var1);
+            break;
+        case 42:
+            ov80_02234BB4(arcadeCtx, frontierMap, var0, var1);
+            break;
+        case 43:
+            ov80_02234DC4(var0, var1);
+            break;
+        case 44:
+            if (var0 == 0) {
+                PaletteData_BlendPalette(frontierMap->paletteData, PLTTBUF_MAIN_OBJ, 0, 0x100, var0, 0);
+            } else {
+                UnkStruct_02239938 *unkStruct = ov80_02239938(ctx->frontierSystem->unk0, var1);
+                u32 palNo = ov42_02229248(unkStruct->unk4);
+                PaletteData_BlendPalette(frontierMap->paletteData, PLTTBUF_MAIN_OBJ, palNo * 0x10, 0x10, var0, 0);
+            }
+            break;
+        case 45:
+            *out = ov80_02235324(arcadeCtx);
+            break;
+        case 46:
+            GfGfx_EngineATogglePlanes(GX_PLANEMASK_BG2, GF_PLANE_TOGGLE_OFF);
+            break;
+        case 47:
+            *out = arcadeCtx->unk1F;
+            break;
+        case 48:
+            *out = 0;
+
+            if (arcadeCtx->type == 0) {
+                if (arcadeCtx->winstreak + 1 == 21) {
+                    *out = 1;
+                } else if (arcadeCtx->winstreak + 1 == 49) {
+                    *out = 2;
+                }
+            }
+            break;
+        case 49:
+            ov80_0222A52C(arcadeCtx->unk330, arcadeCtx->unk314, arcadeCtx->unk31C, arcadeCtx->unk320, NULL, 4, 0xb, 0xcd);
+            break;
+        case 50:
+            ov80_022383C0(arcadeCtx);
+            break;
+        case 51:
+            *out = arcadeCtx->unk1E;
+            arcadeCtx->unk1E = 1;
+            break;
+        case 52:
+            arcadeCtx->unkA7C = 0;
+            break;
+    }
+
+    return FALSE;
+}
+
+static void ov80_02233F1C(FrontierContext *ctx, ArcadeContext *arcadeCtx, u32 a2) {
+    FrontierMap *map = FrontierSystem_GetFrontierMap(ctx->frontierSystem);
+    sub_02015494(ov80_02239A60(map->unk10, 0), a2, ov80_02233F40, arcadeCtx);
+}
+
+extern const VecFx32 ov80_0223BE6C; //{-1, 0, 0}
+static void ov80_02233F40(struct SPLEmitter *emitter) {
+	VecFx16 axis;
```

Comment:
indentation

Reply:
To clarify here: indentation rule is 4 spaces, no tabs